src/components/sidebar-area/app-sidebar-menu-button.tsx

```diff
@@ -42,10 +46,12 @@ export const AppSidebarMenuButton: React.FC<{
             </span>
             <span className='block w-full truncate text-muted-foreground'>
               {data ? (
-                data[data.length - 1].message === DEFAULT_BOT_MESSAGE ? (
+                lastMessage === DEFAULT_BOT_MESSAGE ? (
```

Comment:
Let's avoid double ternaries, could you extract this into it's own component for readability!